makefile

```diff
@@ -0,0 +1,7 @@
+all: black
```

Comment:
The first target, called `all`, here is the one when that starts automatically when you run `make` without specifying a target.